submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/repositories/Repositories.kt

```diff
@@ -79,8 +79,7 @@ interface AccessPermissionRepository : JpaRepository<AccessPermission, Long> {
 }
 
 interface UserDataDataRepository : JpaRepository<UserData, UserDataId> {
-    @Query("FROM UserData ud WHERE ud.userId = :userId AND LOWER(ud.key) LIKE %:dataKey%")
-    fun findByUserAndKey(userId: Long, dataKey: String): List<UserData>
+    fun findByUserIdAndKeyIgnoreCaseContaining(userId: Long, dataKey: String): List<UserData>
```

Comment:
Are we sure about ignoring case here?

Reply:
I wasn't sure about it too but the current code was relying on the service to convert the key to lowercase and this is a native Spring Data method.  In general, accessions are case insensitive so I don't think we should make them case-sensitive for drafts.